packages/pyright-internal/src/analyzer/program.ts

```diff
@@ -84,15 +85,16 @@ import { Scope } from './scope';
 import { getScopeForNode } from './scopeUtils';
 import { IPythonMode, SourceFile } from './sourceFile';
 import { collectImportedByFiles, isUserCode } from './sourceFileInfoUtils';
-import { isStubFile, SourceMapper } from './sourceMapper';
+import { SourceMapper, isStubFile } from './sourceMapper';
```

Comment:
Oh I think the fixall is messing up imports. I probably have to turn it off before saving these.

Reply:
Hmm, I get the same results. Maybe this file wasn't updated after a recent eslint change?